pkg/asset/machines/gcp/machines.go

```diff
@@ -79,7 +79,7 @@ func provider(clusterID string, platform *gcp.Platform, mpool *gcp.MachinePool,
 			Boot:   true,
 			SizeGb: 128,
 			Type:   "pd-ssd",
-			Image:  osImage,
+			Image:  fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/global/images/%s", platform.ProjectID, osImage),
```

Comment:
how is different from the the other pointers to GCP resources, like the network, Subnetwork,

Reply:
Good question. It’s not, so maybe this needs to be fixed at the machine controller. Right now it’s passing in the image value unmodified and failing.